R-packages/evalcast/R/plot_trajectory.R

```diff
@@ -28,7 +28,11 @@
 plot_trajectory <- function(list_of_predictions_cards,
                             first_day = "2020-07-01",
                             last_day = Sys.Date(),
-                            alpha = .2)
+                            alpha = .2,
+                            nrow = 6,
+                            ncol = 4,  
+                           page_facet = 1，
+                           cutoff = 52)
```

Comment:
Ok, last thing I think.

Go through and fix the indenting where you see it.

We want to clean this up so it is easily readable.